src/endpoints/http/collections/export/index.js

```diff
@@ -0,0 +1,143 @@
+import parsers from './lib/parser';
+import archive from './lib/archive';
+import UnsupportedMediaError from '../../../../Errors/UnsupportedMediaError';
+
+const GB = 1073741824; // this will be configurable through fh-config
+
+/**
+ * Gets the size of each collection requested to export.
+ *
+ * @param {db} db - The db connection.
+ * @param {String} collectionName - The name of the collection requested to get the size of.
+ * @returns Promise
+ */
+function getCollectionSizeOne(db, collectionName) {
+  return new Promise((resolve, reject) => {
+    db.collection(collectionName).stats((err, stats) => {
+      if (err) {
+        return reject(err.message === 'ns not found' ? new Error(`${collectionName} collection does not exist`) : err);
+      }
+
+      resolve(stats.size);
+    });
+  });
+}
+
+/**
+ * Gets the size of each collection requested to export.
```

Comment:
You're right, I'll update it. The rest of the explanations for the following functions are wrong too. I'll fix them